RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1344,6 +1313,15 @@ boolean L11_unlockHiddenCity()
 	auto_log_info("Searching for the Hidden City", "blue");
 	if(!in_glover() && !in_tcrs()) 
 	{
+		if(item_amount($item[Stone Wool]) == 0 && have_effect($effect[Stone-Faced]) == 0)
+		{
+			//attempt to summon before using a clover
+			handleFamiliar("item");
```

Comment:
this would run whenever wanting to summon the monster even if it won't happen, and handleFamiliar would lock the regular choice to item familiar,
so annoyingly handling familiar and equipment needs to go either in the individual summon methods or under a new function that checks if the summon will be available

Reply:
created new function `canSummonMonster` to check if we can summon prior to adding to maximizer / picking a fam